localstack-core/localstack/utils/coverage_docs.py

```diff
@@ -1,8 +1,4 @@
-COVERAGE_LINK_BASE = "https://docs.localstack.cloud/references/coverage/"
-MESSAGE_TEMPLATE = (
-    f"API %sfor service '%s' not yet implemented or pro feature"
-    f" - please check {COVERAGE_LINK_BASE}%s for further information"
-)
+_COVERAGE_LINK_BASE = "https://docs.localstack.cloud/references/coverage"
```

Comment:
This link will be replaced by the feature catalog in the future, but for now it's the only thing we have.

/cc @k-a-il for the future, when this will have to change